app/forms/curation_concerns/forms.rb

```diff
@@ -0,0 +1,13 @@
+module CurationConcerns
+ # This will be the module for containing forms
+ # @since 0.14.0
+ # Forms are a type of data structures that include validation that leverage hydra editor
+ # capabilities. Their purpose is to allow users to edit and save objects based on models.
+ # The impetus for their creation was to remove validation from models. A form must define
+ # the model it uses and attributes it manipulates. It may include default values and define
+ # required fields.
+ # They are ruby objects that are typically instantiated by the controller to facilitate
+ # user creation, editing, or deleting of an object.
```

Comment:
I believe my statement about forms reflected my understanding from Sipity (in part inspired by https://github.com/apotonick/reform) in which validations are performed. As I'm reviewing forms in Curation Concerns it appears they are used to sanitize attributes passed along to actors.

I don't know if the "The impetus for their creation was to remove validation from models." holds true within the context of CurationConcerns.

Perhaps @jcoyne or @dchandekstark can weight in?